src/som/interpreter/actors/Actor.java

```diff
@@ -271,7 +267,9 @@ protected void processCurrentMessages(final ActorProcessingThread currentThread,
     private void execute(final EventualMessage msg,
         final ActorProcessingThread currentThread, final WebDebugger dbg, final int i) {
       currentThread.currentMessage = msg;
-      handleBreakPoints(msg, dbg);
+      if (actor instanceof TracingActor) {
```

Comment:
All other code uses checks like `if (VmSettings.ACTOR_TRACING)`, why do you do here something else?